stock_report_quantity_by_location/wizards/stock_report_quantity_by_location.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2019 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import _, fields, models
+
+
+class StockReportByLocationPrepare(models.TransientModel):
+    _name = "stock.report.quantity.by.location.prepare"
+    _description = "Stock Report Quantity By Location Prepare"
+
+    location_ids = fields.Many2many(
+        comodel_name="stock.location", string="Locations", required=True
+    )
+
+    def open(self):
+        self.ensure_one()
+        self._compute_stock_report_by_location()
+        action = {
+            "type": "ir.actions.act_window",
+            "view_mode": "pivot,tree",
+            "name": _("Stock Report by Location"),
+            "context": {
+                "search_default_quantity_gt_zero": 1,
```

Comment:
I didnt see the use of this key. Is this needed ?

Reply:
It use to show only product quantity > 0.